fedn/fedn/utils/plugins/androidhelper.py

```diff
@@ -0,0 +1,108 @@
+import json
+import os
+import tempfile
+
+import numpy as np
+
+from .helperbase import HelperBase
+
+
+class Helper(HelperBase):
```

Comment:
Maybe more a philosophical question, but there seems to be nothing here specific to Android. It is more that we are here assuming that the model parameters have been serialized to a json document, and that that document can be cast to a numpy.array? Maybe add those assumptions to the readme?  I guess what I am after is if it should be named "androidhelper" or something else (jsonhelper?)